.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow creates an OpenSSF scorecard, see https://scorecard.dev/
```

Comment:
Out of curiosity: what are the key differentiaters/benefits compared to the codeql scanning that we currently have in place? 🤓

Reply:
It's rather a "meta-linter" as far as I understand it, e.g. it **checks that** you have CodeQL scans in place.